crossdock/server/yarpc/echo_test.go

```diff
@@ -1,67 +0,0 @@
-package yarpc
```

Comment:
We shouldn't be unit testing things crossdock already tests.

Although to @prashantv's point, that probably should go in another PR.